app/build.gradle

```diff
@@ -126,6 +126,7 @@ android {
             buildConfigField "String", "EVENTLOG_WIKI", "\"commonswiki\""
             buildConfigField "String", "SIGNUP_LANDING_URL", "\"https://commons.m.wikimedia.org/w/index.php?title=Special:CreateAccount&returnto=Main+Page&returntoquery=welcome%3Dyes\""
             buildConfigField "String", "SIGNUP_SUCCESS_REDIRECTION_URL", "\"https://commons.m.wikimedia.org/w/index.php?title=Main_Page&welcome=yes\""
+            buildConfigField "String", "DELETE_CONTRIBUTION_URL", "\"https://commons.wikimedia.org/wiki/Commons:Courtesy_deletions\""
```

Comment:
Opening this link seems inadequate. Check out https://commons.wikimedia.org/wiki/Commons:Deletion_requests